### PR TITLE
new toys for the shield.

### PR DESCRIPTION
**Pull request by cdb-is-not-good**
## About The Pull Request
<details>
<summary>
A very quick 'n sloppy thing for the shield who now have the option of a tweaked lascore with 20% better shot-efficiency and 30% worse firerate. All shield can take it as a primary or you can get it from the teleporter for a bit point cost.. A bit half assed but I've been meaning to add this for a while to help sure up the shields selection of energy weapons.
</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl:

/:cl:

### CODE REVIEW

**Comment by cdb-is-not-good** [comment]
what do you mean "won't fix" @Trilbyspaceclone.....
what did she mean by this u_u

File: code/modules/projectiles/guns/energy/laser/lasercore.dm [hunk 1]
```diff
@@ -54,3 +54,46 @@
 
 	icon_state = iconstring
 	set_item_state(itemstring)
+
+/obj/item/gun/energy/lasercore/militia
+	name = "Blackshield \"Laser Core\" carbine"
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/guns/energy/laser/lasercore.dm:59]:
```suggestion
	name = "Blackshield \"Standard Core\" carbine"
```

File: code/game/objects/items/weapons/storage/blackshield_kits.dm [hunk 2]
```diff
@@ -120,6 +120,20 @@
 		new /obj/item/cell/medium/high(src)
 		new /obj/item/storage/pouch/tubular(src)
 
+/obj/item/storage/box/bs_kit/lascore
+	name = "\improper Lascore Primary Kit"
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/storage/blackshield_kits.dm:124]:
```suggestion
	name = "\improper Standard Primary Kit"
```

File: code/modules/projectiles/guns/energy/laser/lasercore.dm (hunk 1 above)
Comment (cdb-is-not-good) [reply code/modules/projectiles/guns/energy/laser/lasercore.dm:59]: why standard core?

File: code/game/objects/items/weapons/storage/blackshield_kits.dm (hunk 2 above)
Comment (cdb-is-not-good) [reply code/game/objects/items/weapons/storage/blackshield_kits.dm:124]: No. This instantly makes it less clear what it';s supposed to be in its description of the kit.

File: code/modules/projectiles/guns/energy/laser/lasercore.dm [hunk 3]
```diff
@@ -54,3 +54,46 @@
...
+	extra_bulk = 15 //A bit more bulk than an unfolded laser-core.
+	projectile_type = /obj/item/projectile/beam/midlaser //a step down but we're also a good bit more energy efficient.
+	damage_multiplier = 1.1 //essentially the same bonus you get from
+	armor_penetration = 1.2
+	zoom_factors = list(0.4)
+	extra_damage_mult_scoped = 0.2
+	max_upgrades = 3 //we're already pretty beefy.
+	charge_cost = 80
+	fire_delay = 10
+	price_tag = 2500 //SPENSIVE
```
Comment (benj8560) [inline code/modules/projectiles/guns/energy/laser/lasercore.dm:94]:
```suggestion
	price_tag = 2500 //SPENSIVE
	gun_tags = list(GUN_LASER, GUN_ENERGY)
```
caaaan't have a scope slot from parent and built in scope (idk if i can add new lines in a comment about to find out tho)

Comment (benj8560) [reply code/modules/projectiles/guns/energy/laser/lasercore.dm:94]: seems I can cool

**Pull request merged**